framework_tvb/tvb/core/services/project_service.py

```diff
@@ -607,8 +607,8 @@ def _remove_project_node_files(self, project_id, gid, skip_validation=False):
                 specific_remover = get_remover(datatype.type)(datatype)
                 specific_remover.remove_datatype(skip_validation)
                 h5_path = h5.path_for_stored_index(datatype)
-                self.structure_helper.remove_datatype_file(h5_path)
-                encryption_handler.push_folder_to_sync(self.structure_helper.get_project_folder_from_h5(h5_path))
+                self.storage_interface.remove_datatype_file(h5_path)
+                self.storage_interface.push_folder_to_sync(self.storage_interface.get_project_folder_from_h5(h5_path))
```

Comment:
This call could move inside teh above `self.storage_interface.remove_datatype_file(h5_path)`